src/d3a/d3a_core/sim_results/area_statistics.py

```diff
@@ -406,10 +406,10 @@ def _external_trade_entries(child, accumulated_trades):
             spent = round_floats_for_ui(area_data["spentToExternal"][k])
             results["bars"].append({
                 "energy": incoming_energy,
-                "targetArea": child.name,
-                "energyLabel": f"{child.name} bought {abs(incoming_energy)} "
-                               f"kWh from external sources for {k}",
-                "priceLabel": f"{child.name} spent {abs(spent)} cents for {k} to external sources"
+                "targetArea": k,
+                "energyLabel": f"External sources bought {abs(incoming_energy)} "
+                               f"kWh from {k}",
+                "priceLabel": f"External sources spent {abs(spent)} cents"
```

Comment:
You need to check with @sarahhambridge if this is what she really wants here. This label implies that the External sources are buying something from the child, and that the external sources spend money, while it is the other way round, the root area is spending money to buy energy from the external .